packages/ERTP/src/amountMath.js

```diff
@@ -164,3 +179,5 @@ function makeAmountMath(brand, mathHelpersName) {
 }
 
 export default harden(makeAmountMath);
+
+export { makeAmountMath };
```

Comment:
Let's not do this `export default` & `export` unless we are forced into it.

Reply:
Agreed.